test/integration/targets/inventory_docker_machine/playbooks/setup.yml

```
@@ -0,0 +1,26 @@
+---
+- hosts: 127.0.0.1
+  connection: local
+  vars:
+    do_api_token: "{{ lookup('env', 'DIGITALOCEAN_TOKEN') }}"
+    do_region: "{{ lookup('env', 'DIGITALOCEAN_REGION') }}"
+    dm_machine_name: "{{ lookup('env', 'DM_MACHINE_NAME') }}"
+  tasks:
+    - name: Check that Ansible variable do_api_token is set
+      assert: { that: do_api_token != '' }
```

This should be written as:
```suggestion
      assert:
        that:
          - do_api_token != ''
```